src/omv/modules/py_tf.c

```diff
@@ -937,6 +1214,13 @@ STATIC MP_DEFINE_CONST_OBJ_TYPE(
 STATIC const mp_rom_map_elem_t globals_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR___name__),        MP_OBJ_NEW_QSTR(MP_QSTR_tf) },
     #ifdef IMLIB_ENABLE_TF
+    { MP_ROM_QSTR(MP_QSTR_FOMO),                MP_ROM_INT(PY_TF_MODEL_FOMO) },
+    #ifdef IMLIB_ENABLE_TF_YOLO
```

Comment:
You should remove all ifdef/else here and all unavailable functions, and just add a single matching `#endif` at the end of the file to close `#ifdef IMLIB_ENABLE_TF`. If the `IMLIB_ENABLE_TF` is undefined this whole module won't be compiled.